### PR TITLE
feat: prepare for TS defs generation [skip ci]

### LINKED ISSUE #63: Generate TypeScript definitions for P3
Child of vaadin/vaadin-core#186

### PR DESCRIPTION
**Pull request by web-padawan**
Fixes #63 

Generated TS definitions look like this:

```ts
import {PolymerElement} from '@polymer/polymer/polymer-element.js';

import {ThemableMixin} from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';

import {ElementMixin} from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';

import {html} from '@polymer/polymer/lib/utils/html-tag.js';

declare class CookieConsentElement extends
  ElementMixin(
  ThemableMixin(
  PolymerElement)) {
  message: string;
  dismiss: string;
  learnMore: string;
  learnMoreLink: string;
  position: CookieConsentPosition;
  cookieName: string;
  static _finalizeClass(): void;
  connectedCallback(): void;
  disconnectedCallback(): void;
  _show(): void;
}

declare global {
  interface HTMLElementTagNameMap {
    "vaadin-cookie-consent": CookieConsentElement;
  }
}

export {CookieConsentElement};

import {CookieConsentPosition} from '../@types/interfaces';
```

### CODE REVIEW

File: magi-p3-post.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  files: [
+    'package.json',
+    'test/accessibility-test.html',
+    'test/accessibility-test.html',
```
Comment (Haprog) [inline magi-p3-post.js:5]: I guess one of these identical entries could be removed?

File: @types/interfaces.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export type CookieConsentPosition =
+  | 'top'
+  | 'bottom'
+  | 'top-left'
+  | 'top-right'
+  | 'bottom-left'
+  | 'bottom-right';
```
Comment (Haprog) [inline @types/interfaces.d.ts:7]: Is it intentional that this begins with `= | 'top'`?

It looks nice as they are all aligned now on separate lines, just wondering is this a good practice or not. At least TS playground doesn't seem to complain about this. Is this style used elsewhere?

Not a blocker.

File: magi-p3-post.js (hunk 1 above)
Comment (web-padawan) [reply magi-p3-post.js]: Thanks, removed.

File: @types/interfaces.d.ts (hunk 2 above)
Comment (web-padawan) [reply @types/interfaces.d.ts]: I run "format code" in VSCode which is using Prettier. But maybe it's an overkill. Reverted.

**Pull request merged**